android/app/src/main/AndroidManifest.xml

```diff
@@ -1,45 +1,49 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="link.zomtir.cptclient"
-    android:versionCode="1"
-    android:versionName="0.7">
-
-    <!-- Specify the minimum API level -->
-    <uses-sdk android:minSdkVersion="21" />
```

Comment:
No need to set these here anymore, also the versionName and code is updated automatically.